Slim/Schema/Genre.pm

```diff
@@ -35,6 +40,34 @@ use Slim::Utils::Log;
 	$class->resultset_class('Slim::Schema::ResultSet::Genre');
 }
 
+sub loadMyClassicalGenreMap {
+	my $prefs = preferences('server');
+	%myClassicalGenreMap = map {$_ => 1} split(/,\s*/, uc($prefs->get('myClassicalGenres')));
+	$myClassicalGenreIds = undef;
+}
+
+sub isMyClassicalGenre {
+	my $class = shift;
```

Comment:
We could have the same lazy loading of the genre map as we do for the genre IDs: initialise here if needed, rather than call `loadMyClassicalGenreMap` ahead of time?

Reply:
Yes, for consistency. I realised we can't load genre IDs at server initialisation as we can't get the dbh handle at that stage.